app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -107,11 +107,11 @@ $include-html-global-classes: $include-html-classes;
 // $ghost       : #FAFAFA;
 // $snow        : #F9F9F9;
 // $vapor       : #F6F6F6;
-// $white-smoke : #F5F5F5;
+ $white-smoke : #F5F5F5;
```

Comment:
Color `#F5F5F5` should be written as `#f5f5f5`<br>Line should be indented 0 spaces, but was indented 1 spaces